### PR TITLE
Update waybar toggle keybind in readme

### PR DESCRIPTION
**Pull request by rishav12s**
@kRHYME7 another waybar related pr while we are at it

not much of a change or important pr but this can help beginners/newcomers

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -183,7 +183,7 @@ For more information, visit [prasanthrangan/hyde-themes](https://github.com/pras
 | <kbd>Super</kbd> + <kbd>L</kbd> | Launch lock screen |
 | <kbd>Super</kbd> + <kbd>Shift</kbd> + <kbd>F</kbd> | Toggle pin on focused window |
 | <kbd>Super</kbd> + <kbd>Backspace</kbd> | Launch logout menu |
-| <kbd>Ctrl</kbd> + <kbd>Esc</kbd> | Toggle waybar |
+| <kbd>Ctrl</kbd> + <kbd>Alt</kbd> + <kbd>W</kbd> | Toggle waybar |
```
Comment (boulder-roller) [inline README.md:186]: I added this to my PR, thanks for pointing it out. :)

**Review by boulder-roller — COMMENTED** [review]
Can be marked as closed ig

**Comment by rishav12s** [comment]
@boulder-roller I'll close this after your pr gets merged alright ?

**Comment by kRHYME7** [comment]
Will try to prioritize  that one now. Bit busy this days

**Comment by kRHYME7** [comment]
Please test https://github.com/prasanthrangan/hyprdots/pull/1948#pullrequestreview-2425208516 
@rishav12s  @rubiin  @boulder-roller  Thanks!

**Pull request closed without merging**